src/main/java/io/jenkins/plugins/opentelemetry/backend/elastic/ElasticsearchLogsQueryContext-v7.txt

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
ditto

Reply:
kept here to be prepared to rollback or compare if needed.